docs/getting-started/installing-data.md

```diff
@@ -39,3 +39,9 @@ This should output a version number, for example on my machine it shows:
 0.4.1
 ```
 
+\- Get info on data (file or dataset) at path or url
+```bash
+data info http://datahub.io/core/finance-vix
+```
+
+
```

Comment:
Can you have a description of the output here? Maybe you can have output in the console. See this: http://datahub.io/docs/getting-started/pushing-data#push-the-data